public/pages/DetectorsList/List/List.tsx

```diff
@@ -138,8 +146,16 @@ export const DetectorList = (props: ListProps) => {
       search: queryString.stringify(updatedParams),
     });
 
-    dispatch(getDetectorList(GET_ALL_DETECTORS_QUERY_PARAMS));
+    const getUpdatedDetectors = async () => {
+      try {
+        await dispatch(getDetectorList(GET_ALL_DETECTORS_QUERY_PARAMS));
+      } catch (error) {
+        console.log('Error is found during getting detector list', error);
+        setIsLoadingFinalDetectors(false);
+      }
+    };
```

Comment:
This is the only change; the other changes are auto-formatting changes. Somehow my local dev environment stopped auto-formatting on save at some point, fixed it now.